modules/3d/test/ocl/test_tsdf.cpp

```diff
@@ -277,81 +277,6 @@ void renderPointsNormals(InputArray _points, InputArray _normals, OutputArray im
 // ----------------------------
 
 static const bool display = false;
```

Comment:
BTW, tests debugging should use this approach: https://github.com/opencv/opencv/pull/18955